recipe/meta.yaml

```diff
@@ -16,28 +16,29 @@ build:
 requirements:
   host:
     # Drop support for python 3.6 https://github.com/pydata/xarray/pull/4720
-    - python >=3.7
+    - python
     - pip
-    - setuptools >=42
+    - setuptools
     - setuptools_scm_git_archive
-    - setuptools_scm >=3.4
+    - setuptools_scm
     - toml
     - wheel
 
   run:
     - python >=3.7
-    - numpy >=1.17
-    - pandas >=1.0
+    - numpy >=1.18
+    - pandas >=1.1
     # Drop support for python 3.6 https://github.com/pydata/xarray/pull/4720
     - setuptools >=40.4 # For pkg_resources
-    - typing-extensions
+    - typing-extensions >=3.7
 
```

Comment:
Also we need importlib-metadata without preprocess-selector:
`  - importlib-metadata`

https://github.com/pydata/xarray/blob/12a216d632111fc6be39cdaa98335a9bb22f2c5d/setup.cfg#L81
Without this package there will be an error for all users with py37.